protocol/x/clob/ante/clob.go

```diff
@@ -118,6 +120,10 @@ func (cd ClobDecorator) AnteHandle(
 					"Rejected short-term place order with non-zero timeout height < goodTilBlock",
```

Comment:
just to double check, does `ctx` have the address?

Reply:
No I don't think so.

Reply:
Are you asking if `ctx` carries the transaction sender address? I believe not.